src/core/random/SeededRandom.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { SeededRandom } from "./SeededRandom";
+
+describe("SeededRandom", () => {
+    let prng: SeededRandom;
+
+    beforeEach(() => {
+        prng = new SeededRandom("Seed");
+    });
+
+    it("should return a correct single random value", () => {
+        for(let i = 0; i < 1000; i++) {
+            let rand = prng.next();
+            expect(rand).toBeGreaterThanOrEqual(0);
+            expect(rand).toBeLessThanOrEqual(1);
+        }
+    });
+
+    it("should current a correct amount of random bytes", () => {
```

Comment:
should **return** a correct amount of random bytes?

Reply:
Good catch. Will change!

Reply:
Fixed!